libr/bin/pdb/omap.c

```diff
@@ -112,6 +112,14 @@ int omap_remap(void *stream, int address) {
 	// mb (len -1) ???
 	pos = binary_search (omap_stream->froms, address, 0, (len));
 
+	// TODO(nartes): it might introduce seg fault, cause we might peek into
+	// the bytes before allocated area, i.e. omap_stream->froms[-1]
+	//
+	// if (pos == -1)
+	// {
+	//		return -1;
+	// }
+	assert (pos >= 0);
```

Comment:
An assert, you mad?

Reply:
https://travis-ci.org/radare/radare2/jobs/288191151#L864

Reply:
Well, partial compilation didn't desire to run gcc extra time. I thought you had that sort of a macro. Well I'll fix it)